CHANGES.rst

```diff
@@ -267,10 +267,6 @@ astropy.io.ascii
 astropy.io.fits
 ^^^^^^^^^^^^^^^
 
-- Fix bug of dropping table column meta (format, description, meta) when writing a table
```

Comment:
why was this deleted?  It's currently in v3.0.x...?

Reply:
as it was fixing a bug that was also introduced in 3.0.0, see slack discussion

Reply:
nm, I saw a slack message about this (it was changed to afffects-dev when judged to go into 3.0.0 instead of 3.0.1

Reply:
Hah, came to it at the same time...